content/en/js3/blocks/template-html/index.md

```diff
@@ -0,0 +1,69 @@
++++
+title = 'Simplifying element creation'
+headless = true
+time = 15
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Use template tags to simplify element initialisation'
++++
+
+### Using `<template>` tags
+
+We could simplify this code with a different technique for creating elements.
+
+Until now, we have only seen one way to create elements: `document.createElement`. The DOM has another way of creating elements - we can _copy_ existing elements and then change them.
+
+HTML has a useful tag designed to help make this easy, [the `<template>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template). When you add a `<template>` element to a page, it doesn't get displayed when the page loads. It is an inert fragment of future HTML.
+
+We can copy any DOM node, not just `<template>` tags. For this problem, we will use a `<template>` tag because it is designed for this purpose.
+
+When we copy an element, its children get copied. This means we can write our template card as HTML:
```

Comment:
One thing I realise we skip over is that `<template>`, unlike other elements, strips way its `<template>` wrapper when we clone it (which is one of its super powers!) - we should maybe call this out explicitly as a difference...